src/sinks/datadog/logs/config.rs

```diff
@@ -86,7 +86,7 @@ impl DatadogLogsConfig {
         BatchSettings::default()
             .bytes(bytesize::mib(5_u32))
             .events(1_000)
-            .timeout(15)
+            .timeout(1)
```

Comment:
Is this intentional? This is a batch setting, not a rate limit.

Reply:
Ah, no. I've been jumping between branches and an unintentional change got in.

Reply:
Resolved in 0833d4d